packages/core/admin/admin/src/content-manager/components/Wysiwyg/EditorStylesContainer.js

```diff
@@ -1,8 +1,9 @@
 import styled from 'styled-components';
 
 /* eslint-disable */
-/* stylelint-disable */
 export const EditorStylesContainer = styled.div`
+  /* stylelint-disable */
```

Comment:
This line had to be moved into the `styled-components` scope because of the transition from `processors` to `customSyntax`.